server/portal/apps/projects/management/commands/projects_id.py

```diff
@@ -99,29 +75,51 @@ def add_arguments(self, parser):
             '--update',
             action='store',
             type=int,
-            help='Integer to update saved DB value.'
+            help='Update project id DB value.'
+        )
+        parser.add_argument(
+            '--update-using-storage-system-id',
+            action='store_true',
+            help='Update project id DB value using value derived from latest storage system project id.'
+        )
+        parser.add_argument(
+            '--ignore-large-project-ids',
```

Comment:
Similarly, should this be something like `--max-project-id`? Maybe something better

Reply:
Went with max_project_id for this and the related param comment.  still not certain what would be best.